bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -113,34 +112,145 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
+          	<li>Token.services.mozilla.com</li>
+          </ul>
+          <br>
+          <strong>Firefox Hello</strong>
+          <ul>
+          	<li>hello.firefox.com</li>
+          	<li>Loop.services.mozilla.com</li>
+          </ul>
+          <br>
+          <strong>Location (MLS)</strong>
+          <ul>
+          	<li>location.services.mozilla.com</li>
+          	<li>Location-leaderboard.services.mozilla.com</li>
+          </ul>
+          <br>
+          <strong>Marketplace.firefox.com</strong>
+          <ul>
+          	<li>marketplace.firefox.com</li>
+          	<li>receiptcheck.marketplace.firefox.com</li>
+          	<li>Static.marketplace.firefox.com</li>
+          </ul>
+          <br>
+          <strong>Persona</strong>
+          <ul>
+          	<li>browserid.org</li>
+          	<li>firefoxos.persona.org</li>
+          	<li>persona.org</li>
+          	<li>static.login.persona.org</li>
+          	<li>verifier.login.persona.org</li>
+          	<li>www.browserid.org</li>
+          	<li>www.persona.org</li>
+          	<li>yahoo.login.persona.org</li>
+          	<li>gmail.login.persona.org</li>
+          	<li>login.anosrep.org</li>
+          	<li>login.mozilla.org</li>
+          	<li>login.persona.org</li>
+          	<li>Diresworb.org</li>
+          </ul>
+          <br>
+          <strong>Push</strong>
+          <ul>
+          	<li>push.services.mozilla.com</li>
+          	<li>Updates.push.services.mozilla.com</li>
+          </ul>
+          <br>
+          <strong>Telemetry</strong>
+          <ul>
+          	<li>incoming.telemetry.mozilla.org</li>
+          	<li>telemetry-experiment.cdn.mozilla.net</li>
+          	<li>analysis.telemetry.mozilla.org</li>
+          	<li>sql.telemetry.mozilla.org</li>
+          	<li>Metrics.services.mozilla.com</li>
+          </ul>
+          <br>
+          <strong>Test Pilot</strong>
+          <ul>
+          	<li>testpilot.firefox.com</li>
+          </ul>
+          <br>
+          <strong>Tiles</strong>
+          <ul>
+          	<li>tiles.cdn.mozilla.net</li>
+          	<li>tiles.services.mozilla.com</li>
+          </ul>
+          <br>
+          <strong>Tracking Protection</strong>
+          <ul>
+          	<li>shavar.services.mozilla.com</li>
+          	<li>tracking.services.mozilla.com</li>
+          </ul>
+
+          <p>The Mozilla Developer Network (developer.mozilla.org) and wiki.mozilla.org have  been removed from this
+            list due to site vandalism by researchers looking for security issues. They will be
             added back at a later date when a test server is deployed to avoid negative impacts
             on our primary reference site.</p>
 
-          <p>Only Mozilla websites and services are in scope, not email phishing attacks
+          <p>Only Mozilla websites and services are in scope. Issue categories explicitly excluded 
+            from eligibility include not denial of service attacks, email phishing attacks
```

Comment:
Should this read "include denial of service attacks" instead of "include not denial of service attacks"?